fixtures/flight/src/App.js

```diff
@@ -235,9 +235,11 @@ export default async function App({prerender, noCache}) {
           <Foo>{dedupedChild}</Foo>
           <Bar>{Promise.resolve([dedupedChild])}</Bar>
           <Navigate />
-          <React.Suspense fallback={null}>
-            <LargeContent />
-          </React.Suspense>
+          {prerender ? null : ( // TODO: prerender is broken for large content for some reason.
```

Comment:
I'm a bit concerned that this might be a bug in Flight or Fizz and not in the fixture. Do you still want to land it before investigating further?

Reply:
It's an existing issue, whether in the fixture or Flight/Fizz, that is just surfaced by the new addition to the fixture.